### PR TITLE
Update the integration testing readme

### PR DESCRIPTION
**Pull request by waynenilsen**
[rendered](https://github.com/compound-finance/gateway/blob/wayne/update-integration-test-readme/integration/README.md)

### CODE REVIEW

**Comment by jflatow** [comment]
Cool, thanks for trying to keep this up to date

File: integration/README.md [hunk 1]
```diff
@@ -5,23 +5,23 @@ Integration tests that run the real deal and make sure every component works tog
 
 ## Running
 
-First, you'll need to compile Gateway (in release mode) and compile the Ethereum contracts.
+You can run the tests with Gateway compiled in release or debug profile. The default profile is **debug**.
```
Comment (hayesgm) [inline integration/README.md]: This is not true, at least not anymore. Really now we should use `yarn build` and `yarn test` which will do all of the right things all the time.

File: integration/README.md [hunk 2]
```diff
@@ -5,23 +5,23 @@ Integration tests that run the real deal and make sure every component works tog
...
 ```sh
 gateway> cargo +nightly build
 ```
 
-Note: if you require deeper debugging, you may want to enable the `runtime-debug` feature, via:
+Note: if you require deeper debugging, you may want to enable the `integration` feature, via:
 
 ```sh
-gateway> cargo +nightly build --features "runtime-debug"
+gateway> WASM_BUILD_RUSTFLAGS='--cfg feature=\"integration\"' cargo build --release --features integration
```
Comment (hayesgm) [inline integration/README.md]: I feel like these will be fragile versus using `yarn build` from `integration/`

File: integration/README.md [hunk 3]
```diff
@@ -33,12 +33,34 @@ Next, install integration test dependencies in this directory:
...
+```
+
+If you built with the `--release` flag, then run the test using: 
 
 ```sh
-yarn test
+yarn test __tests__/<my-test>.js
 ```
 
+If you did NOT build with the `--release` flag, then run the test using:
```
Comment (hayesgm) [inline integration/README.md]: You can actually just do `PROFILE=debug yarn test ...` I believe.

**Review by hayesgm — CHANGES_REQUESTED** [review]
Awesome thanks; one thing in regards to the new default being `release` instead of debug. Also, possibly just focus on using the `yarn XYZ` variants since they can be kept up-to-date a little easier.

**Pull request merged**